src/sslsocket.cpp

```diff
@@ -336,14 +336,18 @@ enum tls_init_status TLS_init_context(void)
     if (SSL_CTX_use_certificate_file(sip_trp_ssl_ctx,
                                      tls_cert_name,
                                      SSL_FILETYPE_PEM) != 1) {
-        ERROR("TLS_init_context: SSL_CTX_use_certificate_file failed");
+        char errbuf[256] = { '\0' };
+        ERR_error_string_n( ERR_get_error(), errbuf, sizeof(errbuf) );
```

Comment:
Please don't add extra spaces. `ERR_error_string_n(ERR_get_error(), errbuf, sizeof(errbuf));` is perfectly readable, as is `{'\0'};` :)